charts/ai-service/README.md

```diff
@@ -12,11 +12,21 @@ Note that this chart assumes that you have a valid contract with Unique AG and t
 | ---- | ------ | --- |
 | unique-ag |  | <https://unique.ch/> |
 
+## Source Code
+
+* <https://github.com/Unique-AG/helm-charts/tree/main/charts/ai-service>
+
 ## Values
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | affinity | object | `{}` |  |
+| artifactsCache | object | `{"accessMode":"ReadWriteMany","artifacts":[],"enabled":false,"storage":"32Gi","storageClassName":"azurefile"}` | Configuration for artifacts cache |
+| artifactsCache.accessMode | string | `"ReadWriteMany"` | Access mode for artifacts cache. Possible values: ReadWriteOnce, ReadOnlyMany, ReadWriteMany |
+| artifactsCache.artifacts | list | `[]` | List of artifacts to download @param artifactsCache.artifacts[].blobUrl URL of the blob to download @param artifactsCache.artifacts[].path Path where to store the downloaded artifact |
```

Comment:
Nice try Claude with the @param but that does not work in this helm-docs approach :P use simple text ❤️